spec/service_consumers/pact_helper.rb

```diff
@@ -1,9 +1,35 @@
 require "webmock"
 require "pact/provider/rspec"
+require "plek"
```

Comment:
Interesting, I don't see anywhere new `Plek` is being used here.
There are the existing lines on [L123](https://github.com/alphagov/account-api/pull/111/commits/d36ee3d2608ff1c426a15ecdc07c4ec1f4009c9a#diff-7d17ffbce13a8dc8820ff122a86f9a63284e76991d64a9981a7eb7c5697f6bfcR123) and [L124](https://github.com/alphagov/account-api/pull/111/commits/d36ee3d2608ff1c426a15ecdc07c4ec1f4009c9a#diff-7d17ffbce13a8dc8820ff122a86f9a63284e76991d64a9981a7eb7c5697f6bfcR124)


Were those broken previously?

Reply:
Without this. an error gets thrown by the `require "gds_api/test_helpers/email_alert_api"` line:

```
uninitialized constant GdsApi::TestHelpers::EmailAlertApi::Plek
```

So I think:

- Below, where we use `Plek`, the Rails constant autoloading was finding and pulling in the module
- Here, the reference to `Plek` inside the test helper is *not* resolved to `::Plek`, and so it throws an error

Bit weird.